src/Factories/StringService.php

```diff
@@ -42,6 +43,32 @@ public function __construct(string $format, array $dependencies = [])
         $this->format = $format;
     }
 
+    /**
+     * Retrieve a service _as a string_ with the specified name from the given container.
+     *
+     * @param string $serviceName Name of the service to retrieve.
+     * @param ContainerInterface $c The container to retrieve the service from.
+     *
+     * @return string The string representation of the service.
+     *
+     * @throws UnexpectedValueException If service could be converted to string.
+     */
+    protected function resolveString(string $serviceName, ContainerInterface $c): string
```

Comment:
I don't like that this method is duplicated across 2 classes. Perhaps we should add a utility function somewhere that both of these classes can call.

Reply:
Looking at this again and - I'm not sure what I thought I saw - I don't see the other "occurrence" of this code. I must have made a mistake.